new/LIT7033Yelebbesu.xml

```diff
@@ -0,0 +1,80 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7033Yelebbesu" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ይለብሱ፡ አልባሰ፡ ፀርቅ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Yǝlabbǝsu ʾalbasā ḍarq ... (Qǝne of śǝllāse type)</title>
```

Comment:
Maybe here also somehow refer to the authority file?

Reply:
Same as above.